app/components/common/desktopPagination/index.tsx

```diff
@@ -37,11 +38,11 @@ function isLinkPagination(props: DesktopPaginationProps): props is LinkPaginatio
   return (props as LinkPaginationProps).getLinkDestination !== undefined;
 }
 
-function trackActionToClickPagination(actionLabel: string) {
+function trackActionToClickPagination(actionLabel: string, actionArea?: Scinapse.ActionTicket.ActionArea) {
   ActionTicketManager.trackTicket({
     pageType: getCurrentPageType(),
     actionType: "fire",
-    actionArea: "pagination",
+    actionArea: actionArea || "paperList",
```

Comment:
I can't understand why the default actionArea is `paperList` rather than `pagination`

Reply:
The change was proposed by sanghyuk Jung.
If action ticket output `paperList` instead of `pagination` to actionArea, can make a judgment which page has clicked to get more information.

Reply:
👍